lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -1,9 +1,35 @@
+<%namespace name='static' file='/static_content.html'/>
 <%page args="section_data" expression_filter="h"/>
 <%!
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.markup import HTML, Text
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 
+% if configuration_helpers.get_value('ENABLE_SENSITIVE_DATA_MSG_ON_DASHBOARD', settings.FEATURES.get('ENABLE_SENSITIVE_DATA_MSG_ON_DASHBOARD', True)):
+
+<%static:require_module_async module_name="js/vendor/jquery.leanModal" class_name="leanModal">
+
+$( document ).ready(function() {
+  $('[data-section="data_download"]').click(function(){
```

Comment:
The modal is displayed only when the user does click on the tab, but if the user goes directly to the URL `instructor#view-data_download` it does not display the modal. This happens because if I place the code to display the modal after `$( document ).ready(function() {...`  it loads an opacity in all tabs, when I say an opacity, I mean this:

![screenshot from 2018-01-26 11-28-49](https://user-images.githubusercontent.com/3374009/35449847-27b4ff4c-028c-11e8-845d-d5d922a023fe.png)

Note that only loads the opacity but not the modal. 

I'm working on it right now, but question is: It is ok to display the modal only with click event or it's mandatory too when the DOM loads?